via/services/url_details.py

```diff
@@ -21,19 +28,19 @@ def get_url_details(self, url, headers=None):
         :return: 2-tuple of (mime type, status code)
 
         :raise BadURL: When the URL is malformed
+        :raise checkmatelib.BadURL: if the URL is blocked by Checkmate
         :raise UpstreamServiceError: If we server gives us errors
```

Comment:
"If we server"?

Reply:
Fixed this up: https://github.com/hypothesis/via/pull/1009/commits/aeebd90aea2fc3ae4f879cc648d54a751893bf83